packages/celotool/src/cmds/bots/auto-verify.ts

```diff
@@ -185,33 +196,18 @@ async function pollForMessagesAndCompleteAttestations(
       await sleep(POLLING_WAIT)
       continue
     }
-    console.log('')
+    console.info('')
```

Comment:
Seems like we should use either logger or console but not both right?

Reply:
So yeah I was considering taking it out. Basically, in the line above I print out `.` every `POLLING_WAIT` to indicate to a user running this that it is indeed still polling. the `console.info('')` is there to start a new line so that the next log line is a valid JSON line. I'd vote to keep it for that reason